src/Request/Parameters/FormParam.php

```diff
@@ -30,9 +30,6 @@ private function __construct(string $key, $value)
 
     /**
      * Sets encoding header with the key and value provided.
-     *
-     * @param string $key
-     * @param string $value
      */
```

Comment:
Question: Why the parameter documentation is removed in this PR?

Reply:
This was not required in the first place